node/cmd/guardiand/adminserver.go

```diff
@@ -559,12 +559,11 @@ func (s *nodePrivilegedService) SendObservationRequest(ctx context.Context, req
 }
 
 func (s *nodePrivilegedService) GetNextGovernanceVAASequence(ctx context.Context, req *nodev1.GetNextGovernanceVAASequenceRequest) (*nodev1.GetNextGovernanceVAASequenceResponse, error) {
-	maxSequence, err := s.db.MaxGovernanceVAASequence(s.governanceChainId, s.governanceEmitterAddress)
+	nextSequence, err := s.db.NextGovernanceVAASequence(s.governanceChainId, s.governanceEmitterAddress)
```

Comment:
1. It seems WH does not have such data in the database?
2. If different guardians have different db due to bug or network issues, will they produce different next sequence here?

Reply:
>It seems WH does not have such data in the database?

Wormhole also stores governance VAA data, but wormhole does not provide a function similar to `GetNextGovernanceVAASequence`. For governance VAA, the difference between us and Wormhole is that our `Governance` contract and `TokenBridge` contract share the same sequence. I added this function to facilitate getting the next sequence vaa on the guardian.

>If different guardians have different db due to bug or network issues, will they produce different next sequence here?

It will produce a different next sequence if the guardian has a different db. Therefore, the most reliable way to get the next governance sequence is to query the contract state. Maybe we should remove it from the guardian and add a command line to the `clients/js`, WDYT?

Reply:
That's a good approach I think as it reduces Guardian's code